### PR TITLE
[Gekidou] Updated styling opacities

### PR DESCRIPTION
**Pull request by shazm**
#### Summary

Updates the channel and category muted opacity based on https://www.figma.com/file/a9w2q3JnVvBwo2IScP8j3K/Phase-1-Design\?node-id\=4171%3A136108

#### Checklist

- [ ] Added or updated unit tests (required for all new features)
- [X] Has UI changes
- [ ] Includes text changes and localization file updates
- [ ] Have tested against the 5 core themes to ensure consistency between them.

#### Device Information
This PR was tested on: 

#### Screenshots

Screen Shot 2022-05-06 at 10 10 10 am

#### Release Note

```release-note

```

### CODE REVIEW

File: app/components/channel_item/index.ts [hunk 1]
```diff
@@ -40,7 +41,7 @@ const enhance = withObservables(['channel', 'showTeamName'], ({channel, database
             if (!mc) {
                 return of$(false);
             }
-            return observeIsMutedSetting(mc);
+            return observeIsMutedSetting(mc) || of$(isCategoryMuted);
```
Comment (enahum) [inline app/components/channel_item/index.ts:44]: The left side always returns truthy, you would need to pipe it. Also does the category takes presefence over the channel, no?

**Review by enahum — CHANGES_REQUESTED** [review]

File: app/components/channel_item/channel_item.tsx [hunk 2]
```diff
@@ -122,7 +122,7 @@ const ChannelListItem = ({
     const styles = getStyleSheet(theme);
 
     // Make it brighter if it's not muted, and highlighted or has unreads
-    const isBright = !isMuted && (myChannel && (myChannel.isUnread || myChannel.mentionsCount > 0));
+    const isBright = myChannel && (myChannel.isUnread || myChannel.mentionsCount > 0);
```
Comment (enahum) [inline app/components/channel_item/channel_item.tsx:125]: Why was `isMuted` removed?

**Review by enahum — CHANGES_REQUESTED** [review]

File: app/components/channel_item/channel_item.tsx (hunk 2 above)
Comment (shazm) [reply app/components/channel_item/channel_item.tsx:125]: We still want to show the bolder/brighter text even if it's at the new opacity; see https://www.figma.com/file/a9w2q3JnVvBwo2IScP8j3K/Phase-1-Design?node-id=4171%3A136108

With mention, and Unread (first two) still have the original styling - it's just an opacity applied.

**Review by larkox — APPROVED** [review]

**Review by avinashlng1080 — APPROVED** [review]

**Review by enahum — APPROVED** [review]

**Comment by enahum** [comment]
Oh shu... @matthewbirtch I merged this one without realizing you were part of the reviewers, the PR lacked the UX Review label :S 

Feel free to re-open a ticket if you find this not meet the UI specs.

Sorry again.

**Pull request merged**